src/seedu/addressbook/AddressBook.java

```diff
@@ -200,7 +200,18 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        if (args.length >= 2) {
```

Comment:
T2A2 was to improve the code. How is adding a inline method improving the code? Take note to have the correct title for the pull request if not its hard to review.